### PR TITLE
Do object exercise

### PR DESCRIPTION
**Pull request by CBinh710**
# Do Object exercise:
### Task: [Link](https://github.com/users/CBinh710/projects/8/views/1?pane=issue&itemId=80464983)
image

### CODE REVIEW

File: src/exercises/object.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//Object exercise
+const users = [
+    { name: 'Kenny', age: 15 },
+    { name: 'Alice', age: 18 },
+    { name: 'Tom', age: 5 },
+    { name: 'Henry', age: 30 },
+    { name: 'Mike', age: 17 }, 
+  ];
+  // 1. Get all users that their ages are over 18
```
Comment (phuocduong-agilityio) [inline src/exercises/object.js]: Format

File: src/exercises/object.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  const usersOver18 = users.filter(user => user.age > 18);
+  
+  console.log('Get all users that their ages are over 18');
```
Comment (phuocduong-agilityio) [inline src/exercises/object.js]: Please combine 2 console

File: src/exercises/object.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  console.log(usersOver18);
+  
+  // 2. Find user have letter “e” in their name
+  const usersWithE = users.filter(user => user.name.toLowerCase().includes('e'));
```
Comment (phuocduong-agilityio) [inline src/exercises/object.js]: 1. Too specific for naming
2. Create const for magic value. Magic value means value can be changed.

File: src/exercises/object.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  
+  console.log('Users have letter “e” in their name');
+  console.log(usersWithE);
+  
+  // 3. Uppercase people name
+  const usersUpperCase = users.map(user => ({
```
Comment (phuocduong-agilityio) [inline src/exercises/object.js]: Same issue with naming

File: src/exercises/object.js (hunk 1 above)
Comment (CBinh710) [reply src/exercises/object.js]: I updated it, thanks

File: src/exercises/object.js (hunk 2 above)
Comment (CBinh710) [reply src/exercises/object.js]: I did it, thanks

File: src/exercises/object.js (hunk 3 above)
Comment (CBinh710) [reply src/exercises/object.js]: I understood and did it, thanks

File: src/exercises/object.js (hunk 4 above)
Comment (CBinh710) [reply src/exercises/object.js]: I changed it, thanks

File: src/exercises/object.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+// 1. Get all users that their ages are over 18
+
+const MIN_AGE = 18; 
+const usersOver18 = users.filter(user => user.age > MIN_AGE);
+  
+console.log('Get all users that their ages are over 18: ', usersOver18);
+  
+// 2. Find user have letter “e” in their name
+const LETTER_TO_SEARCH = 'e';
+const usersWithE = users.filter(user => user.name.toLowerCase().includes(LETTER_TO_SEARCH));
```
Comment (phuocduong-agilityio) [inline src/exercises/object.js]: UserWithE? Could we change another name?

**Pull request merged**